Source/NSBrowser.m

```diff
@@ -3592,3 +3752,40 @@ - (void) _themeDidActivate: (NSNotification*)notification
 }
 
 @end
+
+@implementation NSBrowser (GSBindingsPrivate)
+
+/* Private methods to handle bindings */
+
+- (void) setValue: (id)anObject forKey: (NSString*)aKey
+{
+  if ([aKey isEqual: NSContentBinding]
+      || [aKey isEqual: NSContentValuesBinding])
+    {
+      // Reload data
+      _passiveDelegate = NO;
+      _itemBasedDelegate = YES;
+
+      [self loadColumnZero];
+      NSDebugLLog(@"NSBinding", @"Setting browser view content/values to %@", anObject);
```

Comment:
This branch seems to ignore the handed in value anObject. Could you please explain how this is supposed to work?

Reply:
This code echoes a similar change in -[NSTableView setValue:forKey:] if NSContentBinding is set, it calls reloadData and ignores the value.   In testing this seems to work with the test harness I have written here:
https://github.com/gcasa/NSBrowser_binding_test.git

Reply:
Given that this is similar to the code in NSTableView, I believe this can be closed.